### DISCUSSION TITLE
What is better 13B q4_0 or 7B q8_0 ???

### DISCUSSION THREAD

**Discussion by skidd-level-100** [ggml-org/llama.cpp#1468] [General]
I would like to know what kind of model is best for its parameter size,
based on file size I would guess that the 13B one is better.

please let me know

**Comment by BetaDoggo** [comment]
In terms of perplexity q4_0 is always better than the 16bit version of the lower model. There are some examples in the readme and there's a [discussion](https://github.com/ggerganov/llama.cpp/discussions/406) with comparisons for each model.
